pkgs/tools/filesystems/idsk/default.nix

```diff
@@ -0,0 +1,33 @@
+
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+
+	pname = "idsk";
+  version = "0.16";
```

Comment:
Where did you find the version number? I'm able to get it from there github.
When then revision is not a release, we have to use `version = "unstable-YYYY-MM-DD" as mentionned there https://nixos.org/nixpkgs/manual/#sec-package-naming

Reply:
It refers to the version display : 
"iDSK version 0.16 (by Demoniak, Sid, PulkoMandy), http://github.com/cpcsdk"
But i will change to unstable-YYYY-MM-DD if it more compliant.